server/cluster.go

```diff
@@ -333,8 +333,9 @@ func (s *Server) bootstrapCluster(clusterID uint64, req *pdpb.BootstrapRequest)
 	cancel()
 	if err != nil {
 		return errors.Trace(err)
-	} else if !resp.Succeeded {
-		return errors.Errorf("bootstrap cluster %d fail, we may not leader", clusterID)
+	}
+	if !resp.Succeeded {
```

Comment:
`if else if` is commonly used in go, I think it is ok, no need to write only if multi times.